src/levanter/utils/hf_utils.py

```diff
@@ -20,3 +25,22 @@ def num_cpus_used_by_tokenizer(tokenizer) -> int:
             return min(max(1, logical_cpu_core_count() - 2), 12)
     else:
         return 1
+
+
+def byte_length_of_token(tokenizer, idx: int) -> int:
+    # this is a pain because we want the prefix spaces, but we don't want extra noise for bytes
```

Comment:
Does this correspond to other implementations that need to compute bpb?  Would be good to reference and comment on whether we're doing the same thing.